ocaml/sdk-gen/go/README.md

```diff
@@ -0,0 +1,74 @@
+# XenServer Go
```

Comment:
The name does not comply with the Go guidelines https://go.dev/brand. We had agreed upon "XenServer SDK for Go".